linter/orb.yaml

```diff
@@ -51,10 +64,16 @@ jobs:
         description: |
           The python version used to run pre-commit.
         type: string
+      pre_commit_version:
```

Comment:
This one is good to keep as-is, since it distinguishes from `python_version` :+1: